gateway/rpc_storage_handler.go

```diff
@@ -808,17 +808,16 @@ func (r *RPCStorageHandler) ProcessKeySpaceChanges(keys []string) {
 
 	for _, key := range keys {
 		splitKeys := strings.Split(key, ":")
-		if len(splitKeys) > 1 && splitKeys[1] == "resetQuota" {
+
```

Comment:
Please add a check here on `splitKeys`  to ensure you are processing a properly formatted key e.g check if the components are 3 or 4. It will be wasteful processing invalid key formats.